src/esp/gfx/RenderingTarget.cpp

```diff
@@ -108,29 +119,53 @@ struct RenderingTarget::Impl {
 
   void readFrameRgba(uint8_t* ptr) {
     framebuffer_.mapForRead(GL::Framebuffer::ColorAttachment{0});
+
+#if defined(CORRADE_TARGET_UNIX) && !defined(CORRADE_TARGET_APPLE)
+    framebuffer_.read(Range2Di::fromSize({0, 0}, framebufferSize_),
+                      colorPixelBuffer_, GL::BufferUsage::StreamRead);
+    auto src_ptr = colorPixelBuffer_.buffer().data();
```

Comment:
The `.data()` does a copy into the CPU memory (and there's another copy inside the GPU memory before that from the FB to the PBO), so I'm not sure if this improves anything :) You might want to `map()` the buffer instead (don't forget to `unmap()` after), but since it's GL the driver might be doing a CPU memory copy underneath anyway.

PBOs should work on Mac without issues as well (or at least magnum's tests claim so), so no need for the `#ifdef` I think.

Reply:
Yeah, I've seen FBO->PBO->CPU be faster than FBO->CPU previously, so I thought I'd see if that was true here also.  Doesn't seem to matter.